setup/setup_fedora.sh

```diff
@@ -0,0 +1,182 @@
+rm lplinux.tar.bz2
+
+# Sublime Text 3
+curl -LO https://download.sublimetext.com/sublime_text_3_build_3143_x64.tar.bz2
+tar xvjf sublime_text_3_build_3143_x64.tar.bz2
+sudo cp -rf sublime_text_3/subime_text.desktop /usr/share/applications/sublime_text.desktop
+echo "You might need to change the icon by pasting Icon=/opt/sublime_text/Icon/128x128/sublime-text.png"
+sudo mv sublime_text_3 /opt/sublime_text
+sudo ln -s /opt/sublime_text/sublime_text /usr/bin/subl
+
+# ngrok https://ngrok.com/
+curl -LO https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip
+unzip ngrok-stable-linux-amd64.zip
+rm ngrok-stable-linux-amd64.zip
+mv ngrok /bin 
+
+# editorconfig for vim
+curl -LO https://github.com/editorconfig/editorconfig-vim/archive/master.zip
+unzip master.zip
+rm master.zip
+mv editorconfig-vim-master ~/.vim
+
+################################################################################
+#                           Customize Shell                                    #
+################################################################################
+
+install_oh_my_zsh () {
+    # Test to see if zshell is installed.  If it is:
+    if [ -f /bin/zsh ] || [ -f /usr/bin/zsh ]; then
+        # Install Oh My Zsh if it isn't already present
+        if [[ ! -d "$dir/oh-my-zsh/" ]]; then
+            sh -c "$(curl -fsSL https://raw.githubusercontent.com/robbyrussell/oh-my-zsh/master/tools/install.sh)"
+        fi
+        # Set the default shell to zsh if it isn't currently set to zsh
+        if [[ ! $SHELL == $(which zsh) ]]; then
+            chsh -s "$(which zsh)"
+        fi
+    else
+        # If zsh isn't installed, get the platform of the current machine
+        platform=$(uname);
+        # If the platform is Linux, try an apt-get to install zsh and then recurse
+        if [[ $platform == 'Linux' ]]; then
+            if [[ -f /etc/redhat-release ]]; then
+                sudo yum install zsh
+            fi
+            if [[ -f /etc/debian_version ]]; then
+                sudo apt-get install zsh
+            fi
+        # If the platform is OS X, tell the user to install zsh :)
+        elif [[ $platform == 'Darwin' ]]; then
+            echo "We'll install zsh, then re-run this script!"
+            brew install zsh
+            exit
+        fi
+    fi
+}
+
+install_oh_my_zsh
+
+###############################################################################
+# Zsh                                                                         #
+###############################################################################
+
+set -P
+# Install Zsh settings
+ln -sf "$PWD/zsh/themes/curiouslearner.zsh-theme" "$HOME/.oh-my-zsh/themes"
+# Zsh Syntax highlighting
+git clone https://github.com/zsh-users/zsh-syntax-highlighting.git "${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-syntax-highlighting"
+
+# Install Powerlevel9k theme
+git clone https://github.com/bhilburn/powerlevel9k.git ~/.oh-my-zsh/custom/themes/powerlevel9k
```

Comment:
For the powerline theme, did you installed the patch fonts anywhere?

How does your terminal looks after setup?

Reply:
https://files.slack.com/files-pri/T0X8CRH5K-F8J9JMWBG/screenshot_from_2017-12-20_20-10-19.png

Reply:
Yups, I installed patch fonts.